core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +218,323 @@ export const border = () => {
+            rounded-10
+          </Text>
+          <Text appearance="default" className="border rounded-15 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-15
+          </Text>
+          <Text appearance="default" className="border rounded-20 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-20
+          </Text>
+          <Text appearance="default" className="border rounded-30 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-30
+          </Text>
+          <Text appearance="default" className="border rounded-40 bg-secondary-lightest p-8 d-inline-block mr-6">
+            rounded-40
+          </Text>
+          <Text appearance="default" className="border rounded-full bg-secondary-lightest p-8 d-inline-block">
+            rounded-full
+          </Text>
+        </div>
+        <div className="mb-5 bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">
+              {
+                '<Text appearance="default" className="border rounded-2-5 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-2-5</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-05 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-05</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-10 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-10</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-15 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-15</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-20 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-20</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-30 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-30</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-40 bg-secondary-lightest p-8 d-inline-block mr-6">rounded-40</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border rounded-full bg-secondary-lightest p-8 d-inline-block">rounded-full</Text>'
+              }
+            </code>
+          </pre>
```

Comment:
Instead of using multiple `<code>` elements, it's better to place all the text within a single `<code>` block.